webui/react/src/pages/DesignKit.tsx

```diff
@@ -202,38 +209,150 @@ const DropdownsSection: React.FC = () => {
-            the option text.
+            By default, the Select truncates option text instead of wrapping to a new line. Because
+            this can lose meaningful information, it is recommended to adjust styles to wrap the
+            option text.
           </li>
         </ul>
       </AntDCard>
       <AntDCard title="Usage">
-        <strong>Default dropdown</strong>
-        <SelectFilter
-          defaultValue={1}
+        <strong>Default Select</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Variations</strong>
+        <strong>Select with default value</strong>
+        <Select
+          defaultValue={2}
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Select with label</strong>
+        <Select
+          label="Select Label"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Select with placeholder</strong>
+        <Select
           options={[
             { label: 'Option 1', value: 1 },
             { label: 'Option 2', value: 2 },
             { label: 'Option 3', value: 3 },
           ]}
+          placeholder="Select Placeholder"
         />
-        <strong>Disabled dropdown</strong>
-        <SelectFilter
+        <strong>Disabled Select</strong>
+        <Select
           defaultValue="disabled"
           disabled
           options={[{ label: 'Disabled', value: 'disabled' }]}
         />
-        <hr />
+        <strong>Select without search</strong>
+        <Select
+          enableSearchFilter={false}
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Search"
+        />
+        <strong>Multiple Select with tags</strong>
+        <Select
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Multiple Select with tags disabled</strong>
+        <Select
+          disableTags
+          mode="multiple"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          value={multiSelectValues}
+          onChange={(value) => setMultiSelectValues(value)}
+        />
+        <strong>Select with tags and default search</strong>
+        <Select
+          enableSearchFilter
```

Comment:
This doesn't seem any different than "Multiple Select with tags". What behavior is changing with this prop?

Reply:
I will remove this variation, I added this to clearly document the difference between the default search and a custom search. But "Multiple Select with tags" does have search so there is nothing new here.